docs/getting_started/sharp-bits.md

```diff
@@ -53,7 +53,7 @@ Array([[0.+0.j, 1.+0.j],
        [1.+0.j, 0.+0.j]], dtype=complex64)
 ```
 
-### Computing the adjoint
+### Computing daggers
```

Comment:
I think we should rephrase this, _daggers_ is not a proper mathematical term.

Reply:
I'll merge this PR and include the change in a small PR with various fixes that's hanging on my local.